make/modules/java.base/Java.gmk

```diff
@@ -23,7 +23,7 @@
 # questions.
 #
 
-DISABLED_WARNINGS_java += this-escape
+DISABLED_WARNINGS_java += this-escape restricted
```

Comment:
I've disabled restricted warnings in java.base as there's a lot of restricted calls in the Linker API implementation :-)